### ISSUE TITLE
Update schedulers to send first Calibration tasks

### ISSUE THREAD

**Issue by teleyinex** [Scifabric/pybossa#373]
BOSSA supported calibration tasks, which are basically tasks for which the creator knows the answer. These tasks could be used for example to qualify the skills of the volunteer or for training users when an application has been doing several batches of tasks and the easiest ones have been already completed. The calibration tasks could be used as reminders. 

As a **user** I want to **progress step by step** in each application.

Basically, the scheduler should check if the application has calibration tasks, and deliver them first, not-respecting the limit of the **n_answers** as these tasks will be used for other purposes and the answer is known for the app creator.
